src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -4336,6 +4336,7 @@ static void beatUnit(XmlWriter& xml, const TDuration dur)
 static void wordsMetronome(XmlWriter& xml, Score* s, TextBase const* const text, const int offset)
 {
     //LOGD("wordsMetronome('%s')", qPrintable(text->xmlText()));
```

Comment:
Could you please remove these comments (and the one in MScoreTextToMXML::writeTextFragments)?

Reply:
@lvinken Alternatively, if you would like to have some debug logging that's not activated by default but can be activated when you need to work on it, you can apply the same idea as in src/framework/audio/internal/audiobuffer.cpp:
```c++
//#define DEBUG_MUSICXML
#ifdef DEBUG_MUSICXML
#define LOG_MXML LOGD
#else
#define LOG_MXML LOGN
#endif

// and then:
LOG_MXML("wordsMetronome('%s')", qPrintable(text->xmlText()));

Reply:
@RomanPudashkin: done. @cbjeukendrup: thanks, good idea, unfortunately no time to implement that now.